dita/passwords.dita

```diff
@@ -229,10 +229,7 @@
 	<topic id="user-facing-services" audience="audience-tech">
 		<title>User facing services</title>
 		<body>
-			<p>Authenticate people accessing user facing services by using the <xref href="https://www.gov.uk/government/publications/introducing-govuk-verify/introducing-govuk-verify" scope="external" format="html">GOV.UK Verify</xref> service.
-			It is not necessary for someone to be a UK Citizen to use the GOV.UK Verify service,
-			but they <ph conref="conrefs.dita#conrefs/policyshall"/> have a UK address.</p>
-			<p>If it is not possible to use GOV.UK Verify, follow the advice presented here to support citizen passwords.
+			<p>Follow the advice presented here to support citizen passwords.
```

Comment:
Is there an agreed approach to use HMG's One Login for new services via Tech Design Authority?

Reply:
Not that I'm aware of. I'll investigate and update this page with a future pull request if required.